dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -198,22 +193,23 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                     {
                         wil::unique_cotaskmem_string unpackagedAppUserModelId;
                         RegisterUnpackagedApplicationHelper(remoteId, unpackagedAppUserModelId);
-                        PushNotificationChannelManager channelManager{};
-                        winrt::PushNotificationChannel pushChannelReceived{ co_await channelManager.CreatePushNotificationChannelForApplicationAsync(unpackagedAppUserModelId.get()) };
+
+                        ChannelDetails channelInfo{};
+                        winrt::hresult hr{ CreateChannelWithRemoteIdHelper(unpackagedAppUserModelId, remoteId, channelInfo) };
 
                         auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
 
                         wil::unique_cotaskmem_string processName;
                         THROW_IF_FAILED(GetCurrentProcessPath(processName));
-                        THROW_IF_FAILED(notificationPlatform->RegisterLongRunningActivator(processName.get()));
+                        THROW_IF_FAILED(notificationPlatform->RegisterLongRunningActivator(processName.get(), s_taskClsid));
```

Comment:
I have a couple of issues with the s_taskClsid:
1) It is a static which should be avoided.
2) I think it can potentially be GUID_NULL for unpackaged apps right? In that case, we need a comment here explaining that.

Reply:
Lets discuss offline